.circleci/config.yml

```diff
@@ -140,14 +140,18 @@ build_and_deploy: &build_and_deploy
             # Example: Merge pull request #35 from eGovPDX/powr-152
             cd /var/www/html
             # Must read the last commit message from Github. The last one in local repo is "CircleCI Deployment"
-            if [[ $(git log origin/master --pretty=format:%s -1) =~ (^Merge pull request #[0-9]* from eGovPDX\/)([^,]*) ]]; then
-              echo "Found a Pull Request Merge for branch: '${BASH_REMATCH[2]}'";
+            last_log=$(git log origin/master --pretty=format:%s -1)
+            lowercase_last_log=`php -r "print strtolower('$last_log');"`
+            if [[ $lowercase_last_log =~ ^(powr-[0-9]+)[[:space:]]+.+$ ]]; then
+              echo "Found the branch name: '${BASH_REMATCH[1]}'";
               # Check if a multidev environment with the branch name exists or not
```

Comment:
I think the term "branch name" is slightly misleading here. Aren't we really referring to a story ID here? I think we should be intentional and consistent about using the right terminology here since a branch name and story ID are sometime, but now always the same.

Reply:
Change the comment to "story ID".